Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
 name = "phoenix-core"
-version = "0.10.0"
+version = "0.11.0-rc.0"
```

Comment:
The version change should be done in a sepparated commit and in the `release-0.11` branch. Not here. This is the procedure we've been following for all the crates.